tests/conftest.py

```diff
@@ -183,8 +188,35 @@ def openreview_context(scope="function"):
         }
 
 
-if __name__ == "__main__":
+@pytest.fixture(scope="session")
+def celery_config():
+    return {
+        # 'broker_url': 'amqp://openreview:openreview@localhost:5672/localhost',
+        "broker_url": "redis://localhost:6379/0",
```

Comment:
can you use 10 as the DB number? when running the integration tests the API uses the DB=0 to use as a cache.

Reply:
Done.